### PR TITLE
[WFLY-17550] Quickstarts Cloud Enhancements

### PR DESCRIPTION
**Pull request by emmartins**
Issue: https://issues.redhat.com/browse/WFLY-17551

### CODE REVIEW

File: quickstarts/WFLY-17551_Quickstarts_cloud_enhancements.adoc [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+
+* https://issues.redhat.com/browse/EAP7-1842 [EAP7-1842]
+* https://issues.redhat.com/browse/WFLY-17289 [WFLY-17289]
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== QE Contacts
+
```
Comment (zhantemirov) [inline quickstarts/WFLY-17551_Quickstarts_cloud_enhancements.adoc:29]:
```suggestion
* mailto:{szhantem@redhat.com}[{Sultan Zhantemirov}]
```

File: quickstarts/WFLY-17551_Quickstarts_cloud_enhancements.adoc [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+== Issue Metadata
+
+=== Issue
+
+* https://issues.redhat.com/browse/WFLY-17551[WFLY-17551]
+
+=== Related Issues
+
+* https://issues.redhat.com/browse/EAP7-1842 [EAP7-1842]
```
Comment (zhantemirov) [inline quickstarts/WFLY-17551_Quickstarts_cloud_enhancements.adoc:21]:
```suggestion
* https://issues.redhat.com/browse/EAP7-1842[EAP7-1842]
```

File: quickstarts/WFLY-17551_Quickstarts_cloud_enhancements.adoc [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+* https://issues.redhat.com/browse/WFLY-17289 [WFLY-17289]
```
Comment (zhantemirov) [inline quickstarts/WFLY-17551_Quickstarts_cloud_enhancements.adoc:22]:
```suggestion
* https://issues.redhat.com/browse/WFLY-17289[WFLY-17289]
```

**Comment by emmartins** [comment]
suggested changes done @zhantemirov , thank you

**Pull request closed without merging**